extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ObjectQueryService.java

```diff
@@ -220,6 +220,9 @@ public String generateQuery(String filter,
                         logger.warn("Given a referral, but referrals are disabled. Error was: {}", e.getMessage());
                         logger.debug("Got a referral, but configured to not follow them.", e);
                     }
+                } catch (LDAPException e) {
+                  logger.warn("Failed to process an LDAP search result.");
+                  logger.error("Error processing LDAPEntry search result.",e);
```

Comment:
Logging an exception in this way will produce a stacktrace in the logs. This is good, but should only be done at the debug level. The is the origin for the pattern you see above:

https://github.com/apache/guacamole-client/blob/b3f3b18f0bbd063b4f9ce8ce50329ac3f1e47148/extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ObjectQueryService.java#L220-L221

The idea is to not confront users/administrators with stacktraces unless they truly are interested in debugging information.

Reply:
Oops, I meant for that to be in the debug logs. The warning should be sufficient to point the admin in the right direction.  I've moved the stack trace to debug.